src/typescript/frontend/src/components/StatsBar.tsx

```diff
@@ -1,32 +1,32 @@
 import { Listbox } from "@headlessui/react";
 import { ChevronDownIcon } from "@heroicons/react/20/solid";
-import { useState } from "react";
+import Link from "next/link";
 
 type Props = {
   marketNames: string[];
+  selectedMarket: string;
 };
 
-export function StatsBar({ marketNames }: Props) {
-  const [selectedMarket, setSelectedMarket] = useState<string>(marketNames[0]);
-
+export function StatsBar({ marketNames, selectedMarket }: Props) {
```

Comment:
Hmmm I usually go with defining the props type separately - the above format is fine with two fields but it can get unwieldy pretty quickly as you add more.

You can see the prop type when you hover over the type, which I find is easier to parse anyways?

<img width="703" alt="Screenshot 2023-05-25 at 23 07 08" src="https://github.com/econia-labs/econia/assets/127273034/cff77dea-e5dd-4829-84eb-890732adffd2">